plugins/module_utils/waiters.py

```diff
@@ -299,6 +299,42 @@
                 },
             ]
         },
+        "NatGatewayDeleted": {
+            "delay": 5,
+            "maxAttempts": 40,
+            "operation": "DescribeNatGateways",
+            "acceptors": [
+                {
+                    "state": "success",
+                    "matcher": "pathAll",
+                    "expected": "deleted",
+                    "argument": "NatGateways[].State"
+                },
+                {
+                    "state": "success",
+                    "matcher": "error",
+                    "expected": "NatGatewayNotFound"
+                }
+            ]
+        },
+        "NatGatewayAvailable": {
+            "delay": 5,
+            "maxAttempts": 40,
+            "operation": "DescribeNatGateways",
+            "acceptors": [
+                {
+                    "state": "success",
+                    "matcher": "pathAll",
+                    "expected": "available",
+                    "argument": "NatGateways[].State"
+                },
+                {
+                    "state": "success",
```

Comment:
it should be "retry" here, I think

Reply:
@s-hertel You are right! Just changed! When copying and pasting I forget to change. Thank you.